packages/EasyStandard/tests/Rector/ExplicitBoolCompareRector/Fixture/MethodCallWithShortTrueComparison.php.inc

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Tests\Rector\ExplicitBoolCompareRector\Fixture;
+
+class MethodCallWithShortTrueComparison
+{
+    public function isValid(): bool
+    {
+        return true;
+    }
+
+    public function testProcess(): void
+    {
+        if ($this->isValid()) {
+            echo 'valid';
+        }
+    }
+}
+
+?>
+
+-----
```

Comment:
We don't need the second part for correct cases.

Reply:
Done.